spec/event_store_client/store_adapter/api/client_spec.rb

```diff
@@ -94,9 +96,56 @@ module EventStoreClient::StoreAdapter::Api
       end
 
       it 'sends a correct request' do
-        api_client.consume_feed('stream_name', 'subscription_name', count: 10, long_poll: 5)
+        api_client.consume_feed(stream_name, 'subscription_name', count: 10, long_poll: 5)
+        expect(stub).to have_been_requested
+      end
+    end
+
+    describe '#link_to' do
+      let(:event_1) do
+        EventStoreClient::Event.new(
+          id: SecureRandom.uuid,
+          title: '3@Stream',
+          type: 'SomethingHappened'
+        )
+      end
+      let(:event_2) do
+        EventStoreClient::Event.new(
+          id: SecureRandom.uuid,
+          title: '2@OtherStream',
+          type: 'SomethingElseHappened'
+        )
+      end
+
+      let(:body) do
+        events.map do |event|
+          {
+            eventId: event.id,
+            eventType: '$>',
+            data: event.title
+          }
+        end
+      end
+
+      let!(:stub) do
+        stub_request(:post, stream_url(stream_name)).
+          with(body: body.to_json)
+      end
+
+      it 'sends a correct request' do
+        api_client.link_to(stream_name, events)
         expect(stub).to have_been_requested
       end
     end
+
+    private
```

Comment:
Private inside of `Rspec.describe?` does it have effect?

Reply:
Honestly have no clue but I was always put custom methods in spec under private.